docs/zh_settings.html

```diff
@@ -59,6 +59,8 @@ <h3><img src="assistant.png" alt="Assistant"/> &nbsp; <a id="assistant" class="n
 
       <p>
         &nbsp; &nbsp; 使用这个表单来设置屏幕捕获；您可以配置两次<strong> 屏幕捕获 </strong>（又名时钟滴答，即tick）之间的时间（以毫秒为单位）和灰度级数。
+          从 Dayon 开始！15 开始，也可以使用 sRGB 颜色作为替代。在这种情况下，灰度级数将被忽略。
```

Comment:
@rmshadows  may I ask you to have a brief look at the Chinese grammar please?

    zh_settings.html
    Babylon_zh.properties

Very appreciated, many thanks in advance!